### PR TITLE
added field to match configured state

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-api/terraform/template/cognito.tf [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-  name           = "efcms-${var.environment}"
+  name = "efcms-${var.environment}"
 
   auto_verified_attributes = ["email"]
 
   username_attributes = ["email"]
 
   verification_message_template {
-    default_email_option = "CONFIRM_WITH_LINK"
+    default_email_option  = "CONFIRM_WITH_LINK"
```
Comment (matthopson) [inline web-api/terraform/template/cognito.tf]: Estra space.

Comment (matthopson) [reply web-api/terraform/template/cognito.tf]: (or is that intentional?)

File: web-api/terraform/template/cognito.tf [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
     email_message_by_link = "Please click the link below to verify your email address. {##Verify Email##} "
     email_subject_by_link = "U.S. Tax Court account verification"
   }
 
+  sms_authentication_message = "{####}"
```
Comment (matthopson) [inline web-api/terraform/template/cognito.tf:14]: Can we change this to `¯\_(ツ)_/¯`? :P

**Review by matthopson — APPROVED** [review]

File: web-api/terraform/template/cognito.tf (hunk 2 above)
Comment (sutt0n) [reply web-api/terraform/template/cognito.tf:14]: this was the field that was different, so we sync'd it to match and it fixed it somehow. ¯\_(ツ)_/¯

File: web-api/terraform/template/cognito.tf (hunk 1 above)
Comment (sutt0n) [reply web-api/terraform/template/cognito.tf]: apparently my editor wants that extra space there...

Comment (sutt0n) [reply web-api/terraform/template/cognito.tf]: @matthopson removed the extra space. i run `terraform` for vscode and it has an autoformatter. ¯\_(ツ)_/¯ 

marking this as resolved

**Pull request merged**